remoto/log.py

```diff
@@ -12,7 +12,9 @@ def reporting(conn, result, timeout=None):
         try:
             received = result.receive(timeout)
             level_received, message = list(received.items())[0]
-            log_map[level_received](message.strip('\n'))
+            message = message.splitlines()
```

Comment:
Hi alfredodeza,
Thanks a lot for the comments.
I did some tests to validate the code change.
Test 1: change message.strip('\n') to message.strip('\n^M'), the change does not work. The log refers to below:
[2016-06-20 02:25:10,510][arm-vm1][DEBUG ] Selecting previously unselected package ceph-common.^M
[2016-06-20 02:25:10,577][arm-vm1][DEBUG ] (Reading database ... ^M(Reading database ... 5%^M(Reading database ... 10%^M(Reading database ... 15%^M(Reading database ... 20%^M(Reading database ... 25%^M(Reading database ... 30%^M(Reading database ... 35%^M(Reading database ... 40%^M(Reading database ... 45%^M(Reading database ... 50%^M(Reading database ... 55%^M(Reading database ... 60%^M(Reading database ... 65%^M(Reading database ... 70%^M(Reading database ... 75%^M(Reading database ... 80%^M(Reading database ... 85%^M(Reading database ... 90%^M(Reading database ... 95%^M(Reading database ... 100%^M(Reading database ... 125715 files and directories currently installed.)^M
[2016-06-20 02:25:10,586][arm-vm1][DEBUG ] Preparing to unpack .../ceph-common_10.2.2-1xenial_arm64.deb ...^M

Test 2: change message.strip('\n') to message.strip('\n\r'). The result is better than test1. There is still ^M in the log. The log refers to below:
[2016-06-20 02:35:07,418][arm-vm1][DEBUG ] Selecting previously unselected package ceph-common.
[2016-06-20 02:35:07,483][arm-vm1][DEBUG ] (Reading database ... ^M(Reading database ... 5%^M(Reading database ... 10%^M(Reading database ... 15%^M(Reading database ... 20%^M(Reading database ... 25%^M(Reading database ... 30%^M(Reading database ... 35%^M(Reading database ... 40%^M(Reading database ... 45%^M(Reading database ... 50%^M(Reading database ... 55%^M(Reading database ... 60%^M(Reading database ... 65%^M(Reading database ... 70%^M(Reading database ... 75%^M(Reading database ... 80%^M(Reading database ... 85%^M(Reading database ... 90%^M(Reading database ... 95%^M(Reading database ... 100%^M(Reading database ... 125715 files and directories currently installed.)
[2016-06-20 02:35:07,492][arm-vm1][DEBUG ] Preparing to unpack .../ceph-common_10.2.2-1xenial_arm64.deb ...
Test3: change message.strip('\n') to 
            message = message.splitlines()
            for i in range(len(message)):
                log_map[level_received](message[i].strip("'"))
The log refers to below:
[2016-06-20 02:38:26,602][arm-vm1][DEBUG ] Selecting previously unselected package ceph-common.
[2016-06-20 02:38:26,667][arm-vm1][DEBUG ] (Reading database ...
[2016-06-20 02:38:26,668][arm-vm1][DEBUG ] (Reading database ... 5%
[2016-06-20 02:38:26,668][arm-vm1][DEBUG ] (Reading database ... 10%
[2016-06-20 02:38:26,669][arm-vm1][DEBUG ] (Reading database ... 15%
[2016-06-20 02:38:26,669][arm-vm1][DEBUG ] (Reading database ... 20%
[2016-06-20 02:38:26,670][arm-vm1][DEBUG ] (Reading database ... 25%
[2016-06-20 02:38:26,670][arm-vm1][DEBUG ] (Reading database ... 30%
[2016-06-20 02:38:26,671][arm-vm1][DEBUG ] (Reading database ... 35%
[2016-06-20 02:38:26,671][arm-vm1][DEBUG ] (Reading database ... 40%
[2016-06-20 02:38:26,671][arm-vm1][DEBUG ] (Reading database ... 45%
[2016-06-20 02:38:26,672][arm-vm1][DEBUG ] (Reading database ... 50%
[2016-06-20 02:38:26,673][arm-vm1][DEBUG ] (Reading database ... 55%
[2016-06-20 02:38:26,673][arm-vm1][DEBUG ] (Reading database ... 60%
[2016-06-20 02:38:26,677][arm-vm1][DEBUG ] (Reading database ... 65%
[2016-06-20 02:38:26,677][arm-vm1][DEBUG ] (Reading database ... 70%
[2016-06-20 02:38:26,678][arm-vm1][DEBUG ] (Reading database ... 75%
[2016-06-20 02:38:26,678][arm-vm1][DEBUG ] (Reading database ... 80%
[2016-06-20 02:38:26,679][arm-vm1][DEBUG ] (Reading database ... 85%
[2016-06-20 02:38:26,679][arm-vm1][DEBUG ] (Reading database ... 90%
[2016-06-20 02:38:26,679][arm-vm1][DEBUG ] (Reading database ... 95%
[2016-06-20 02:38:26,680][arm-vm1][DEBUG ] (Reading database ... 100%
[2016-06-20 02:38:26,680][arm-vm1][DEBUG ] (Reading database ... 125715 files and directories currently installed.)
[2016-06-20 02:38:26,689][arm-vm1][DEBUG ] Preparing to unpack .../ceph-common_10.2.2-1xenial_arm64.deb ...
In the 3rd log, there is no ^M any more. 
in the above three code changes, the 3rd can remove all the ^M, it works well.